bindings/c/foundationdb/fdb_c.h

```diff
@@ -91,12 +91,21 @@ extern "C" {
     DLLEXPORT WARN_UNUSED_RESULT fdb_error_t fdb_add_network_thread_completion_hook(void (*hook)(void*), void *hook_parameter);
 
 #pragma pack(push, 4)
+#if FDB_API_VERSION >= 700
```

Comment:
Since this is PR only for master whose API version is supposed to 700 at-least, isn't this case always going to be true in present as well as in future, so this case may not be necessary as if we ever compile an older version it wont have this commit anyway.

Reply:
The API version for `fdb_c.h` is not actually fixed like it is for most of our other files. Instead, a user that includes it is required to specify an API version, and by doing so they are effectively selecting the API that the C library provides. So for example, a C program might look like:

```
#define FDB_API_VERSION 620
#include <foundationdb/fdb_c.h>
```

Based on the above, the user would get the v620 API for any version of `fdb_c.h` greater than or equal to 6.2, including this one.